submissions/sprint_b/helpers.py

```diff
@@ -0,0 +1,180 @@
+import os
+import json
+import random
+import unittest
+
+class FileManager:
+    """Handle local File System IO"""
+
+    @staticmethod
+    def get_extension(path):
+        """Get extension from file path"""
+        return os.path.splitext(path)[-1][1:]
+
+    @staticmethod
+    def read_json(path, mode="r", *args, **kwargs):
+        with open(path, mode=mode, *args, **kwargs) as handle:
+            return json.load(handle)
+
+
+class Vocabulary:
+    """Standardize vocabulary representation from multiple sources."""
+
+    files = FileManager()
+
+    @classmethod
+    def from_file(cls, path, *args, **kwargs):
+        extension = cls.files.get_extension(path)
+        representation = cls.strategies(extension)(path, *args, **kwargs)
+
+        return representation
+
+    @classmethod
+    def from_json(cls, path, fields=True, *args, **kwargs):
+        data = cls.files.read_json(path, *args, **kwargs)
+        if fields:
+            representation = (data, data.keys())
+        else:
+            representation = data
+        print representation
+        return representation
+
+    @classmethod
+    def strategies(cls, file_extension, intent='read'):
+        input_strategies = {"json": cls.from_json}
+        if intent is "read":
+            return input_strategies[file_extension]
+
+class EpithetGenerator:
+    """Pull from the reasources folder to generate the vocabulary and then use the
+    vocabulary to generate epithets"""
+
+
+    @classmethod
+    def vocab_generator(cls):
+        file_manager = FileManager()
+        vocabulary = Vocabulary()
+        resources = ['/Users/Ben/Documents/Kenzie/Code/Activities/PYTHON/backend-epithet-generator/resources/data.json']
+        for path in resources:
+            extension = file_manager.get_extension(path)
+            if extension == 'json':
+                epi_dict = vocabulary.from_file(path)
+                return epi_dict
+            else:
+                vocabulary.from_file(path)
+
+    @classmethod
+    def epi_generator(cls):
+        epi_dict = cls.vocab_generator()
+        return (random.choice(epi_dict[0]["Column 1"]) + " "
+                + random.choice(epi_dict[0]["Column 2"]) + " "
+                + random.choice(epi_dict[0]["Column 3"]))
+
+
+epithet_generator = EpithetGenerator()
+
+class TestFileManager(unittest.TestCase):
+    """Test the FileManager class within the helpers file"""
```

Comment:
Test classes should not be defined in the same file as the code. Seperate these into the test_helpers.py file.